services/QuillLMS/spec/models/concerns/user_cacheable_spec.rb

```diff
@@ -240,5 +257,52 @@
       expect(post_update_first_page_fetch).to eq(3)
       expect(post_update_second_page_fetch).to eq(4)
     end
+
+    context 'cache invalidation' do
+      let(:teacher) { create(:teacher, :with_classrooms_students_and_activities) }
+      let!(:old_value) { teacher.all_classrooms_cache(key: 'test.key') { false } }
```

Comment:
What about extracting out `let(:key) { 'test.key' }` to make explicit the coupling that exists to each of the tests.

Reply:
Seems like a super-reasonable change.  I'll do that.